### PR TITLE
aar update 6

### PR DESCRIPTION
**Pull request by adrian-kong**
per yaml here: https://drive.google.com/drive/folders/1zQ5vSVevyu5i3l37Yy9dvs9pS2Wh04fR

### CODE REVIEW

File: settings.yaml [hunk 1]
```diff
@@ -1410,6 +1420,72 @@
...
+  type: integer
+  units: Solution Period
+  default value: '50'
+  readonly: false
+  enumerated possible values:
+  Description: Number of Solution Periods between AAR PNM PCCF messages being sent.
+  Notes:
+
+- group: nmea
+  name: pcsd_msg_rate
```
Comment (adrian-kong) [inline settings.yaml:1479]: resolves https://snav.slack.com/archives/C020JLK6PK8/p1694661289356779

File: settings.yaml [hunk 2]
```diff
@@ -2145,7 +2274,7 @@
   Description: Maximum storage device usage.
   units: percent
   Notes: Sets a limit on how full the storage device can be before logging is stopped. If the drive is more than this percent full, no new log files will be created and a warning will be logged every 30 seconds.
-         If this setting is changed while logging is enabled, it will go into effect on the next file that’s created.
+    If this setting is changed while logging is enabled, it will go into effect on the next file that’s created.
```
Comment (adrian-kong) [inline settings.yaml:2277]: unsure if this affects notes. indents are removed in the yaml.

File: settings.yaml [hunk 3]
```diff
@@ -3252,7 +3390,7 @@
   expert: false
   type: enum
   units: N/A
-  default value: 'SBP (Swift Binary Protocol)'
+  default value: 'SBP'
```
Comment (adrian-kong) [inline settings.yaml:3395]: to follow enumerated possible value list

**Review by switanis — APPROVED** [review]
LGTM

**Pull request merged**